samples/nrf9160/nrf_cloud_mqtt_multi_service/overlay_nrfcloud_logging.conf

```diff
@@ -18,9 +18,8 @@ CONFIG_LOG_BUFFER_SIZE=4096
 # Select which format to generate. Dictionary logs are compact but require
 # decoding on the server or user PC.  Text logs are verbose but immediately
 # visible in the nRF Cloud web portal.
-CONFIG_LOG_BACKEND_NRF_CLOUD_OUTPUT_DICTIONARY=y
-#CONFIG_NRF_CLOUD_LOG_DICT_JSON=y
-#CONFIG_LOG_BACKEND_NRF_CLOUD_OUTPUT_TEXT=y
+#CONFIG_LOG_BACKEND_NRF_CLOUD_OUTPUT_DICTIONARY=y
```

Comment:
while you are in these files:
https://github.com/nrfconnect/sdk-nrf/commit/9af0e3d2dc0673e7638621b9b2ea020e1c4eafc6#r116082219

Reply:
Thanks, I missed your comment.

Reply:
```
CONFIG_NRF_CLOUD_LOG_LEVEL_DBG=y
CONFIG_NRF_CLOUD_LOG_LOG_LEVEL_INF=y
```
is that what you want for the overlay?

Reply:
This one I'll remove: CONFIG_NRF_CLOUD_LOG_LEVEL_DBG=y

This one sets the log level for the cloud logging system: CONFIG_NRF_CLOUD_LOG_LOG_LEVEL_INF=y.  I'd like to keep it that way.